spec.bs

```diff
@@ -947,11 +959,11 @@ A <dfn for=fencedframetype>destination event</dfn> is either a
        beacons](https://wicg.github.io/attribution-reporting-api/#ref-for-request-service-workers-mode).
 
     : [=request/origin=]
-    :: <span class=XXX>Get the origin from somewhere, like the [implementation
-       does](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:content/browser/fenced_frame/fenced_frame_reporter.cc;l=384;drc=183dd920ea5d6aed92e007c40e226f1f840c0567)</span>.
+    :: |request initiator| if |event| is a [=fencedframetype/destination URL event=],
+       |destination info|'s [=reporting destination info/reporting url declarer origin=] otherwise.
 
     : [=request/referrer=]
-    :: `"no-referrer"`
+    :: |request initiator|
```

Comment:
This is also a big change. Do we really perform zero referrer policy redaction?

Reply:
If by redaction you're referring to the "referrer policy", we do set it to ["origin"](https://w3c.github.io/webappsec-referrer-policy/#referrer-policy-origin) in the impl, which should be reflected in the spec (I'll add that in). Or is this referring to something else?

Reply:
Redaction as in allow a referrer policy to limit the contents of whatever "referrer" is eventually present on the request. "no-referrer" means the https://w3c.github.io/webappsec-referrer-policy/ performs no redaction

Reply:
I had forgotten to update that. We redact it to [just show the origin](https://source.chromium.org/chromium/chromium/src/+/main:content/browser/fenced_frame/fenced_frame_reporter.cc;l=699;drc=5b6ee2e0c7eca5b5c5c26f59349f4c1daeb7de1e), rather than having zero redaction. It should be reflected in the spec now (and in the preview once I figure out why it's not updating).